scripts/build.sh

```diff
@@ -120,7 +122,7 @@ function download_file() {
     echo "... downloaded $NAMESPACE to $DESTINATION"
 }
 
-if [[ $(basename $("pwd")) != "kelp" ]]
+if [[ $(basename $("pwd")) != "kelp" && $("pwd") != *"circleci/project"* ]];
```

Comment:
Why add the circleci exception now when it wasn’t needed earlier?

Reply:
There was a very specific reason for this I think its the way the modules install and the new directory structure but it was ages ago and I have forgotten the details :laughing:

Reply:
Ok no worries. Let’s leave it in, if you remember the reason then please add that context as a comment next to the code for posterity :)